tests/python_tests/test_llm_pipeline.py

```diff
@@ -846,8 +846,9 @@ def test_pipelines_with_gguf_generate(pipeline_type, model_ids):
 
 @pytest.mark.parametrize("pipeline_type", get_gguf_pipeline_types())
 @pytest.mark.parametrize("model_ids", get_gguf_model_list())
+@pytest.mark.parametrize("enable_save_ov_model", [False])
```

Comment:
why?

Reply:
Test purpose only, try to figure out why MacOS test failed (segment fault): **https://github.com/openvinotoolkit/openvino.genai/actions/runs/15420607456/job/43434100193**

It seems macos-13 has smaller memory size: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Does it make sense to create a separate test case, instead of check 3 pipeline test (HF/GGUF/OV native) to only 2 pipeline test (GGUF/OV native) to save memory usage?

Reply:
@mryzhov, please advice other MacOS runner with more memory

Reply:
test results shows that not related to insufficient memory, https://github.com/openvinotoolkit/openvino.genai/actions/runs/15436902324/job/43447403549?pr=2218, will revert back to `macos-13` and continue to investigation.

Reply:
@akladiev provides my a local macos machine w/ macos-12 + 64GB memory usage (during test the memory usage ~30GB + 15GB Swap usage), cannot reproduce the same segment fault issue.
 
I found the same issue in other PR：[Bump actions/download-artifact from 4.1.9 to 4.3.0 · openvinotoolkit/openvino.genai@c3fe438](https://github.com/openvinotoolkit/openvino.genai/actions/runs/15438377512/job/43451509230?pr=2315), so I think it not introduced by my PR, could you please help to further investigate this issue?

Reply:
I try to separate `enable_save_ov_model` related gguf test from original test case to reduce memory usage. 
Still the macos test failed with https://github.com/openvinotoolkit/openvino.genai/blob/3c232ac2761c8068d1a45892df3db8b69802fa47/tests/python_tests/test_llm_pipeline.py#L909
GHA action: https://github.com/openvinotoolkit/openvino.genai/actions/runs/15485048274/job/43598941161